diesel/src/mysql/connection/bind.rs

```diff
@@ -70,3 +155,25 @@ fn mysql_type_to_ffi_type(tpe: MysqlType) -> ffi::enum_field_types {
         MysqlType::Blob => MYSQL_TYPE_BLOB,
     }
 }
+
+fn known_buffer_size_for_ffi_type(tpe: ffi::enum_field_types) -> Option<libc::c_ulong> {
+    use std::mem::size_of;
+    use self::ffi::enum_field_types;
```

Comment:
Fine. You win this ti— wait! Not so fast! You can still `use self::ffi::enum_field_types as t;` to reduce all the visual noise ;)